server/base/src/main/java/org/apache/accumulo/server/security/delegation/AuthenticationTokenSecretManager.java

```diff
@@ -282,6 +284,7 @@ protected SecretKey generateSecret() {
     return super.generateSecret();
   }
 
+  @SuppressFBWarnings(value = "HSM_HIDING_METHOD", justification = "")
```

Comment:
This was marked as hiding `SecretManager.createSecretKey`. I'm wondering if we need this method at all.

Reply:
Looked into this and I don't think it can be removed. `SecretManager` is a Hadoop class (so can't change scope) and `createSecretKey` is a protected method within that. So, can access it in this class (extends `SecretManager`) but not in the class that calls `AuthenticationTokenSecretManager.createSecretKey()`